test_elasticsearch_serverless/test_server/test_rest_api_spec.py

```diff
@@ -568,20 +568,25 @@ def remove_implicit_resolver(cls, tag_to_remove):
         # Each file may have a "test" named 'setup' or 'teardown',
```

Comment:
This comment should also mention the `requires` section.

Reply:
Thanks, addressed in [`545a1cc` (#48)](https://github.com/elastic/elasticsearch-serverless-python/pull/48/commits/545a1cce8d831dfab1d7be4246c3db85414c4650)